Publish paola

paolap opened the pull request:
I tried to improve the issue with the publish-options page looking bad

1) I moved Zenodo, Figshare.. out of CLEX and gave it its own "generalist repositories" title. I added a sentence at the start of the page to clarify this.

2) I created tabs for each group: BoM, CSIRO, CLEX, Generalist repositories and Discipline repositories.
<img width="1290" alt="Example with TABS" src="https://user-images.githubusercontent.com/4155949/173991754-65ccda22-7121-4e6b-965b-dce7c289c10e.png">

3) Move ESGF after NCI-geonetwork

4) Updated publish-intro as the text didn't match anymore the actual guidelines

Apart from these I also edited the publish-zenodo page

Finally this is a question:
We have in the book appendix a list of journal editors and their publishing requirements, we should somehow link them to this guidelines. 
As we also have a page with dataset examples at the end of the guidelines, we should be consistent. Either put both in the book appendix or create a kind of guideline appendix and move the journals requirements at the end of the guidelines too
Any idea?

AliciaTak reviewed (approved):
It looks good to me Paola. I haven't had a detailed read of the book, but it makes sense to move the dataset examples to the appendix, too.

hot007 commented on Governance/publish/publish-intro.md:
```diff
@@ -1,14 +1,15 @@
+These guidelines are covering the main steps involved in publishing a climate dataset.
+We tried to list the publishing options available to climate researchers in Australia, based on their organisation and the type of data to publish. 
+We then covered some of these options in detail, highlighting how to prepare a dataset for publication, what to include both in the dataset itself and in the metadata record.
+We focused on institutions and projects we know of that publish climate data. If you are aware of others that we didn't cover please contact us by creating an issue.
```
link [creating an issue](https://github.com/ACDguide/Governance/issues/new/choose) in this sentence.

paolap replied:
about the journal/dataset examples I agreed they should either both in the appendix or at the end of this chapter, which might make more sense, but they're basically equally relevant. And I'll add the link, thanks for spotting that

hot007 replied:
Yep I'd be happy for it to be moved in here (ie update toc), feel free to add it to this PR if possible.

hot007 commented on Governance/publish/publish-options.md:
```diff
@@ -2,41 +2,55 @@
-## BoM
+::::{tab-set}
+:::{tab-item} BoM 
+Insert here pathways for BoM researchers...
```
Request review/contribution from someone at BoM - who?

paolap replied:
I think Avi was happy to contribute and Griffith (?) mentioned they're publishing with C3S so he could contribute that

hot007 reviewed (approved):
Looks good @paolap , nice re-ordering. I made one small edit and suggested addition of a link to create issues.

Regarding you question about page/appendix consistency, I actually wonder if that page of journal requirements isn't at least as relevant as the exemplar datasets page in this chapter - I would support moving the journal requirements page from appendices to this chapter, depending what you and others think?

paolap commented:
I've added the issue link requested by Claire plus a few more for the portals listed in publish-options (as TERN, CDS etc). I've also moved the journal list from the appendix to these guidelines. Can easily be moved back. I don't think anything in here breaks the main and two people reviewed it so @chloemackallah if you're busy it's still probably safe to merge it. Should you hate the restructure it's not too difficult to revert.

chloemackallah reviewed (approved):
Looks great, I like the tabs a lot!
